sim/riscof/Makefile

```diff
@@ -0,0 +1,53 @@
+###############################################################################
+#
+# Copyright 2020 OpenHW Group
+# Copyright 2023 Dolphin Design
```

Comment:
If this is new and modified, please add "Copyright 2024 Silicon Labs".

Reply:
Added the copyright now

Reply:
@MikeOpenHWGroup do any of the other files need such a header too?

Reply:
Any _new_ file must have a copyright and license header, and any file that is _meaningfully modified_ under the preexisting license should have a new copyright declared by the contributor or contributor's organization.

Reply:
There are a few python files, ld scripts and header files that are added that don't have a copyright and license as they are generated and modified based on riscof output. I can add the appropriate headers to those.